content/en/docs/tasks/administer-cluster/declare-network-policy.md

```diff
@@ -66,27 +66,32 @@ pod/nginx-701339712-e0qfq   1/1           Running       0          35s
 
 ## Test the service by accessing it from another pod
 
-You should be able to access the new `nginx` service from other pods. To test, access the service from another pod in the default namespace. Make sure you haven't enabled isolation on the namespace.
+You should be able to access the new `nginx` Service from other Pods. To test, access the Service from another Pod in the `default` namespace. Make sure you haven't enabled isolation on the namespace.
 
-Start a busybox container, and use `wget` on the `nginx` service:
+You can start a busybox container to get a shell and use `wget` on the `nginx` Service:
 
 ```console
 kubectl run --generator=run-pod/v1 busybox --rm -ti --image=busybox -- /bin/sh
 ```
 
-```console
-Waiting for pod default/busybox-472357175-y0m47 to be running, status is Pending, pod ready: false
+```none
+If you don't see a command prompt, try pressing enter.
+```
 
-Hit enter for command prompt
+In your shell, run the following `wget` command:
 
+```console
 / # wget --spider --timeout=1 nginx
```

Comment:
This makes more sense as:
```suggestion
wget --spider --timeout=1 nginx
```

and in that case you can mark the block as `shell` rather than `console`.

Reply:
@sftim you suggestions look good. Before I make the changes is there an authentic/official way to establish the minimum version required other than looking at the change log / code, etc.

Reply:
> is there an authentic/official way to establish the minimum version required other than looking at the change log / code, etc.

If there is @sharjeelaziz I have yet to find it.
The official answer is usually to consult the website. Of course, SIG Docs don't get that luxury.

Sometimes you can glean details by searching https://github.com/kubernetes/enhancements/ or https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/

Reply:
Thanks @sftim that is what I thought. Let me dig through and find out.

Reply:
Done.